### PR TITLE
Updates to denote community supported drivers

### PR DESCRIPTION
**Pull request by rpaik**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: docs/content/Configuration/Connecting-to-the-Database.mdx [hunk 1]
```diff
@@ -10,9 +10,12 @@ redirect_from:
...
 
 <InfoBox>
-  If you'd like to connect to a data store which is not yet supported or present on this page,
+  If you'd like to connect to a data store which is not yet listed on this page,
   please <a href="https://github.com/cube-js/cube.js/issues">file an issue</a> on GitHub.
-  You can also contribute a driver, see this <a href="https://github.com/cube-js/cube.js/blob/master/CONTRIBUTING.md#implementing-driver">step-by-step guide</a> for guidelines.
+  You can also contribute a driver, see this <a href="https://github.com/cube-js/cube.js/blob/master/CONTRIBUTING.md#implementing-driver">step-by-step guide</a> for guidelines.<br/><br/>
+  
+  <b>A note on driver support:</b><br/>
+  Most of the drivers for data sources on this page are supported either directly by Cube Dev or via individual vendors. Rest are community supported and those drivers will be listed under the <em>Community supported</em> section.
```
Comment (adnanrahic) [inline docs/content/Configuration/Connecting-to-the-Database.mdx:18]:
```suggestion
  Most of the drivers for data sources on this page are supported either directly by Cube Dev or via individual vendors. The rest are community supported and those drivers will be listed under the <em>Community supported</em> section. 
```

**Review by adnanrahic — APPROVED** [review]
Just a small nitpick, but this looks great!

File: docs/content/Configuration/Connecting-to-the-Database.mdx (hunk 1 above)
Comment (rpaik) [reply docs/content/Configuration/Connecting-to-the-Database.mdx:18]: Thanks @adnanrahic 😄

**Comment by rpaik** [comment]
@igorlukanin how do you like the infobox I added for Hive (https://github.com/cube-js/cube.js/pull/4738/files#diff-b22e4f202c90f404901d46b92da43306fc1da2c6866861caced480f59667ee13)? 

If you're OK with this, I'll replicate this for other community drivers.

**Pull request merged**